src/documentation.html

```diff
@@ -42,6 +42,7 @@ <h2>Setup</h2>
     {% endfor %}
   </div>
   <h2>Multiprotocol</h2>
+  <h5>Running Zigbee and Thread on the same wireless chip</h5>
```

Comment:
The h5 doesn't look very nice, IMHO. This is probably default browser style. We have a hint class we use on the first Getting started page, maybe we can reuse this?

```suggestion
  <div class="text-center hint">Running Zigbee and Thread on the same wireless chip</div>
```

Reply:
I see your point. But what hint does, is make it italic and centered. Updated it to be italic, but not centered.